tests/unit/type/hash.tcl

```diff
@@ -482,6 +482,13 @@ start_server {tags {"hash"}} {
         list [r hincrby htest foo 2]
     } {2}
 
+    test {HINCRBY HINCRBYFLOAT against non-integer increment value} {
+        r del incrhash
+        r hset incrhash field 5
+        assert_error "*value is not an integer*" {r hincrby myhash field v}
+        assert_error "*value is not a*" {r hincrbyfloat myhash field v}
```

Comment:
Typo: `incrhash` is populated but `myshash` is tested.

Reply:
Thanks @zuiderkwast , I have updated latest diff with corrected key name.